Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.19-alpine as builder
+FROM golang:1.21-alpine as builder
```

Comment:
It would be great if you could upgrade go version to 1.23

Reply:
Done